packages/components/inputs/localized-rich-text-input/src/rich-text-input.tsx

```diff
@@ -1,14 +1,57 @@
-/* eslint-disable react/prop-types */
-import { PureComponent } from 'react';
-import PropTypes from 'prop-types';
+import { PureComponent, type ReactNode } from 'react';
 import pick from 'lodash/pick';
-import requiredIf from 'react-required-if';
+import type { Value } from 'slate';
+// TODO: remove after upgrade of `slate-react` to the latest version
+// @ts-ignore
 import { Editor } from 'slate-react';
```

Comment:
The wrapper component also needs to be defined. See https://github.com/commercetools/ui-kit/pull/2129/files#diff-c5851815d5a8e6ae99dfe4d2298aa2c3c9a45f8642a3c9d43b09afed31e59ccfR33-R37

Reply:
I'll take care of it and merge the PR afterwards

Reply:
Alright, thanks 👍🏽